javascript/javascript1/week2/event.js

```diff
@@ -0,0 +1,23 @@
+//Event application---<>-------<>-------<>-
+
+const weekdays = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+function getEventWeekday(numberOfDay) {
+  // Declare function passing in how many days event will be held
+  const d = new Date(); // getting the date (object- inbuilt class)
+  let today = d.getDay(); // getting the current day in number
```

Comment:
you can call this todayNumber or merge this two line like:
`const today = new Date().getDay()`